datalad_xnat/init.py

```diff
@@ -143,10 +143,32 @@ def __call__(url, project=None, force=False, dataset=None):
         lgr.info('XNAT reports %i subjects currently on-record for project %s',
                  nsubj, project)
 
+        # check if dataset already initialized
+        auth_dir = ds.pathobj / '.datalad' / 'providers'
+        if auth_dir.exists() and not force:
+            yield dict(
+                res,
+                status='error',
+                message='Dataset found already initialized, '
+                        'use `force` to reinitialize',
+            )
+            return
+
         # put essential configuration into the dataset
-        config.set('datalad.
+        config.set('datalad.xnat.default.url',url,where='dataset')
+        config.set('datalad.xnat.default.project',project,where='dataset')
```

Comment:
Style comment: There should be a space after each comma in a function call.

Reply:
Done